arreglo/src/main/java/com/globant/model/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.globant.model;
+
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue (strategy = GenerationType.AUTO)
+    private Integer id;
+    private String name;
+
```

Comment:
Order should have a @ManyToOne relationship with Items and also a @JoinColumn with a foreign key defintion and a referenced column name